packages/openneuro-app/src/scripts/dataset/comments/coral-embed.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useRef, useState } from "react"
+
+declare global {
+  interface Window {
+    Coral?: {
+      createStreamEmbed: (config: any) => void
+    }
+  }
+}
```

Comment:
Doing this is fine but we should probably move this to a global definitions typescript file near the entrypoint for the project, it can be surprising elsewhere.